### ISSUE TITLE
Can't run RimPy on Linux

### ISSUE THREAD

**Issue by nearlyNonexistent** [rimpy-custom/RimPy#86]
![image](https://user-images.githubusercontent.com/4041565/185812434-35f9c208-d00e-4387-9763-26b7a3ee2dc6.png)

No logs folder exists as it can't get far enough to make it. My Arch system is fully updated as of the time of posting, as well. Using the most recent prerelease.

**Comment by nvvtype** [comment]
I'm having the exact same error, but I'm using the workaround provided by the mod author.

```
me@mycomputer ~/D/RimPy_Linux [1]> bash -c 'LD_PRELOAD="/usr/lib/libgio-2.0.so.0:/usr/lib/libglib-2.0.so.0" ./RimPy'
./RimPy: symbol lookup error: /usr/lib/libgio-2.0.so.0: undefined symbol: g_module_open_full

```

i've looked up various other workarounds but none of them have done anything so far, this is where I'm getting stuck.  Shame because now I have to do mod load order... manually... again...

**Comment by nanomoeamezao** [comment]
same issue here on Manjaro

**Comment by rimpy-custom** [comment]
Closing for now. Reopen if issue wasn't fixed with latest Linux build 1.2.6.28.
